go.mod

```diff
@@ -4,10 +4,12 @@ go 1.21
 
 toolchain go1.21.1
 
+replace github.com/celestiaorg/quantum-gravity-bridge => github.com/sweexordious/quantum-gravity-bridge v1.4.1-0.20230913173701-eebfd521b5ba
+
 require (
-	github.com/celestiaorg/celestia-app v1.0.0-rc14
+	github.com/celestiaorg/celestia-app v1.0.0-rc9.0.20230913154707-424f63163e42
```

Comment:
[note to reviewers]
Similar

Reply:
[023dadc](https://github.com/celestiaorg/orchestrator-relayer/pull/469/commits/023dadc38b01730aa8a3e3f7609846c289b51bf0)